php-frontend/src/main/java/org/sonar/php/parser/PHPGrammar.java

```diff
@@ -1645,21 +1643,8 @@ public MemberAccessTree OBJECT_MEMBER_ACCESS() {
         b.firstOf(b.token(ARROW), b.token(NULL_SAFE_ARROW)),
         b.firstOf(
           VARIABLE_WITHOUT_OBJECTS(),
-          OBJECT_DIMENSIONAL_LIST(),
-          NAME_IDENTIFIER_OR_KEYWORD())));
-  }
-
-  public ExpressionTree OBJECT_DIMENSIONAL_LIST() {
-    return b.<ExpressionTree>nonterminal(PHPLexicalGrammar.OBJECT_DIM_LIST).is(
```

Comment:
I think we can also drop `PHPLexicalGrammar.OBJECT_DIM_LIST`.

Reply:
Done. Thanks for noticing!